source/hook-osx.cpp

```diff
@@ -56,6 +56,8 @@ static pthread_cond_t hook_control_cond;
 static pthread_mutex_t pressed_keys_mutex;
 static pthread_mutex_t released_keys_mutex;
 
+int hook_status = 0;
```

Comment:
better be to initialize it as failure UIOHOOK_FAILURE

Reply:
sure, good idea

Reply:
done